make/test.mk

```diff
@@ -127,6 +127,53 @@ e2e-cleanup:
 	oc login -u system:admin
 	oc delete project devconsole-e2e-test || true
 
+.PHONY: test-olm-integration
```

Comment:
Could we move this to a bash script and call it from here?

Reply:
We will split it into smaller multiple targets.